tests/unit/models/package-info-cache-test.js

```diff
@@ -226,23 +241,60 @@ describe('models/package-info-cache.js', function() {
     });
   });
 
-  describe('discoverProjectAddons tests on external-dependency project', function() {
-    beforeEach(function() {
-      projectPath = path.resolve(addonFixturePath, 'external-dependency');
-      packageJsonPath = path.join(projectPath, 'package.json');
-      packageContents = require(packageJsonPath);
-      project = new Project(projectPath, packageContents, ui, cli);
-      project.discoverAddons();
+  describe('discoverProjectAddons', function() {
+    let fixturifyProject;
 
-      pic = project.packageInfoCache;
-      projectPackageInfo = pic.getEntry(projectPath);
+    afterEach(function() {
+      if (fixturifyProject) {
+        fixturifyProject.dispose();
+      }
     });
 
-    it('shows dependency packages exist and are in project.addonPackages', function() {
-      expect(project.addonPackages).to.exist;
-      let packageNames = Object.keys(project.addonPackages);
-      expect(packageNames.length).to.equal(9);
-      expect(packageNames.indexOf('ember-cli-blueprint-test-helpers')).to.be.above(-1);
+    describe('within an addon', function() {
```

Comment:
should this simply be unified with the instantiation tests?

Reply:
It doesn't actually require the instantiation tests, since the original setup in the test data (which you've replicated in some way in fixturify, though I haven't worked out the exact structure) did function correctly without instantiation.